ui/app/AppMain.qml

```diff
@@ -539,14 +539,23 @@ Item {
                 // Whole this Connection object should be moved to the nim side.
                 // Left here only cause we don't have a way to deal with translations on the nim side.
 
+                const title = isContact ? qsTrId("contact-request-accepted") :
+                                          //% "New contact request"
+                                          qsTrId("new-contact-request")
+
+                const message = //% "You can now chat with %1"
+                              isContact ? qsTrId("you-can-now-chat-with--1").arg(Utils.removeStatusEns(name)) :
+                                          //% "%1 requests to become contacts"
+                                          qsTrId("-1-requests-to-become-contacts").arg(Utils.removeStatusEns(name))
+
+                if (Qt.platform.os === "linux") {
+                    // Linux Notifications are not implemented in Nim/C++ yet
+                    return systemTray.showMessage(title, message, systemTray.icon.source, 4000)
+                }
```

Comment:
Since we seem to keep doing this check over and over again, wouldn't it make sense to move this logic into `root.store.showOsNotification()` ?

Reply:
We could, but as far as what I can see from the TODOs, this code is gonna be moved to the Nim side soon(?). Also, I guess we might want to implement the Linux implementation of OS notifications in DotherSide someday.
All that for me to say that this code is not permanent and shouldn't change much, so might as well leave it as is for now.